pkg/reconciler/accessor/networking/certificate.go

```diff
@@ -44,7 +44,7 @@ func ReconcileCertificate(ctx context.Context, owner kmeta.Accessor, desired *v1
 
 	recorder := controller.GetEventRecorder(ctx)
 	if recorder == nil {
-		return nil, fmt.Errorf("recoder for reconciling Certificate %s/%s is not created", desired.Namespace, desired.Name)
+		return nil, fmt.Errorf("recorder for reconciling Certificate %s/%s is not created", desired.Namespace, desired.Name)
```

Comment:
I'm always worried about changes like his, but as the line above is `recorder`, I'm relatively confident here